tests/testmodels.py

```diff
@@ -637,3 +637,26 @@ class Principal(Model):
     school: fields.OneToOneRelation[School] = fields.OneToOneField(
         "models.School", on_delete=fields.CASCADE, related_name="principal", to_field="id"
     )
+
+
+class Post(Model):
+    id = fields.IntField(pk=True)
+    name = fields.TextField()
+
+    comments = fields.ReverseRelation["Comment"]
+    favorites = fields.ReverseRelation["Favorite"]
+
+
+class Comment(Model):
+    id = fields.IntField(pk=True)
+    content = fields.TextField()
+    post: fields.ForeignKeyRelation[Post] = fields.ForeignKeyField(
+        "models.Post", related_name="comments"
+    )
+
+
+class Favorite(Model):
+    id = fields.IntField(pk=True)
+    post: fields.ForeignKeyRelation[Post] = fields.ForeignKeyField(
+        "models.Post", related_name="favorites"
+    )
```

Comment:
I think adding new models is not essential. Also Current test model is enough huge. How about reusing `DoubleFK` model instead?

Reply:
I think models we need are "multiple backward relation" not "multiple foriegn key relation" that is your suggestion. So, how about using just `Tournament` that has two backward relation?
(And your model redundant model removing opinion is acceptable. - Done)

Reply:
I misunderstand point. Anywhere change looks good :+1: